matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/widgets/DefaultWidgetURLFormatter.kt

```diff
@@ -53,11 +52,10 @@ internal class DefaultWidgetURLFormatter @Inject constructor(private val integra
         val preferredConfig = integrationManager.getPreferredConfig()
         if (!this::currentConfig.isInitialized || preferredConfig != currentConfig) {
             currentConfig = preferredConfig
-            val defaultWhiteList = stringProvider.getStringArray(R.array.integrations_widgets_urls).asList()
-            whiteListedUrls = when (preferredConfig.kind) {
-                IntegrationManagerConfig.Kind.DEFAULT    -> defaultWhiteList
-                IntegrationManagerConfig.Kind.ACCOUNT    -> defaultWhiteList + preferredConfig.restUrl
-                IntegrationManagerConfig.Kind.HOMESERVER -> listOf(preferredConfig.restUrl)
+            whiteListedUrls = if (matrixConfiguration.integrationWidgetUrls.isEmpty()) {
+                listOf(preferredConfig.restUrl)
+            } else {
+                matrixConfiguration.integrationWidgetUrls
```

Comment:
This code is not equivalent (no use of `kind` anymore)? Maybe it's ok, just wanted to mention it

Reply:
Make same logic than iOS and web